commit e40213450b53157967a1f83eda50e9a941c13a08
Author: Ying Huang <huang.ying.caritas@gmail.com>
Date:   Tue May 18 14:35:14 2010 +0800

    ACPI, APEI, EINJ support
    
    EINJ provides a hardware error injection mechanism, this is useful for
    debugging and testing of other APEI and RAS features.
    
    Signed-off-by: Huang Ying <ying.huang@intel.com>
    Signed-off-by: Andi Kleen <ak@linux.intel.com>
    Signed-off-by: Len Brown <len.brown@intel.com>

diff --git a/drivers/acpi/apei/Kconfig b/drivers/acpi/apei/Kconfig
index d61c851533c9..5f0a41c2bc62 100644
--- a/drivers/acpi/apei/Kconfig
+++ b/drivers/acpi/apei/Kconfig
@@ -6,3 +6,11 @@ config ACPI_APEI
 	  to the operating system. This improves NMI handling
 	  especially. In addition it supports error serialization and
 	  error injection.
+
+config ACPI_APEI_EINJ
+	tristate "APEI Error INJection (EINJ)"
+	depends on ACPI_APEI && DEBUG_FS
+	help
+	  EINJ provides a hardware error injection mechanism, it is
+	  mainly used for debugging and testing the other parts of
+	  APEI and some other RAS features.